tests/test_watchmedo.py

```diff
@@ -48,3 +48,25 @@ def test_load_config_invalid(tmpdir):
         watchmedo.load_config(yaml_file)
 
     assert not os.path.exists(critical_dir)
+
+
+def make_dummy_script(tmpdir, n=10):
+    script = os.path.join(tmpdir, 'auto-test-%d.py' % n)
+    with open(script, 'w') as f:
+        f.write('import time\nfor i in range(%d):\n\tprint("+++++ %%d" %% i, flush=True)\n\ttime.sleep(1)\n' % n)
+    return script
+
+
+def test_kill_auto_restart(tmpdir, capfd):
```

Comment:
Maybe replacing `capfd` with `capsys` would help on Windows :thinking:

Reply:
Allegedly capfd is required to capture the subprocess output. I have to leave it for a while. Tried running the subprocess directly in a cmd.exe terminal and the stderr doesn't show there. It would in linux. The other asserts definitely show that the stop is working so that part is OK.